### ISSUE TITLE
mapfishapp - own contexts manager UI

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#1601]
Rennes Métropole would like to have a UI in mapfishapp so that users may manage (see, download, retrieve permalink, delete) their own contexts.

This feature would be reserved to logged in users only.

Currently, contexts are saved in db (see [schema](https://github.com/georchestra/georchestra/blob/15.12/postgresql/02-mapfishapp.sql)) each time the user:
 * creates a permalink
 * downloads the current map context WMC

The expected UI is a grid showing fields: 
* `file_hash` (aka uuid), 
* `created_at`, 
* `last_access`, 
* `access_count`, 
* and a bunch of fields extracted from WMC string (ie from field `raw_file_content`), namely: 
  * `title` (`/ViewContext/General/Title`), 
  * `abstract` (`/ViewContext/General/Abstract`), 
  * possibly `keywords` (`/ViewContext/General/KeywordList`) and
  * `bbox` and `srs` (`/ViewContext/General/BoundingBox`).

**Comment by fvanderbiest** [comment]
Expected JSON for the `/mapfishapp/ws/geodocs/wmc` endpoint:
```json
{
    "contexts": [{
        "hash": "7408588b0727ea99b8d410221b6b6e82",
        "created_at": "2016-12-07 07:57:01.042346",
        "last_access": "2016-12-13 08:34:59.638648",
        "access_count": 524,
        "title": "test title",
        "abstract": "test abstract",
        "keywords": ["kw1", "kw2"],
        "srs": "EPSG:2154",
        "bbox": [323939.570888990012, 6267251.48890830018, 324663.650888990029, 6267642.36890830006]
    }, {
        "hash": "aba8c547aa3c9418df31a1fb2bf6da7b",
        "created_at": "2016-12-06 11:45:08.894042",
        "last_access": "2016-12-06 11:45:09.035564",
        "access_count": 1,
        "title": "test title 2",
        "abstract": "test abstract 2",
        "keywords": ["kw3"],
        "srs": "EPSG:2154",
        "bbox": [323939.570888990012, 6267251.48890830018, 324663.650888990029, 6267642.36890830006]
    }]
}
```

The `srs` and `bbox` fields stem from 
```xml
<BoundingBox minx="360768.312889139983" miny="6394235.33127340022" maxx="396972.312889139983" maxy="6415151.33127340022" SRS="EPSG:2154" />
```

**Comment by fvanderbiest** [comment]
Current state of dev
![capture du 2017-01-03 15 57 07](https://cloud.githubusercontent.com/assets/265319/21611515/70a861d0-d1cd-11e6-9833-18a382f51de7.png)

**Comment by fvanderbiest** [comment]
Merged into master for 16.12
